docs/api_v2/api_v2_reference.apib

```diff
@@ -3163,60 +3543,85 @@ The user account is identified by the provided authentication token.
     - githubUsername:    `torvalds.`                                             (string)
     - websiteUrl:        `http://www.example.com`                                (string)
     - twitterUsername:   `@BillGates`                                            (string)
-- proxy                                       (object, required)
-    - password:          `ad78knd9Jkjd86`                                        (string, required)
-    - groups                                                                  (array, required)
-- limits                 (object, required)
-    - maxParallelCrawlers:  `10`        (number, required)
-    - monthlyPages:         `900000`    (number, required)
-    - monthlyComputeUnits:  `300`       (number, required)
-    - dataRetentionDays:    `7`         (number, required)
-    - maxMemoryMbytes:      `16384`     (number, required)
-    - maxActsCount:         `100`       (number, nullable)
-    - maxActs2Count:        `100`       (number, nullable)
-- currentBillingPeriod    (object, required)
-    - startAt:              `2018-03-14T23:00:00.000Z`        (string, required)
-    - endAt:                `2018-04-14T21:59:59.000Z`        (string, required)
-    - usage                                              (object, required)
-        - pagesCrawled:     `900000`                        (number, required)
-        - computeUnits:     `300`                           (number, required)
+- id:               `YiKoxjkaS9gjGTqhF`                     (string, required)
+- email:            `bob@example.com`                       (string, required)
+- currentBillingPeriod                                      (object, required)
+    - startAt:              `2018-03-14T23:00:00.000Z`      (string, required)
+    - endAt:                `2018-04-14T21:59:59.000Z`      (string, required)
+    - usage                                                 (object, required)
+        - pagesCrawled:     900000                          (number, required)
+        - computeUnits:     300                             (number, required)
+- proxy                                             (object, required)
+    - password:          `ad78knd9Jkjd86`           (string, required)
+    - groups                                        (array, required)
+- limits                                            (object, required)
+    - monthlyPages:                  900000         (number, required)
+    - monthlyComputeUnits:           300            (number, required)
+    - monthlyResidentialProxyGbytes: 0              (number, nullable) # TODO: review
+    - monthlyGoogleSerpRequests:     20             (number, nullable) # TODO: review
+    - dataRetentionDays:             7              (number, required)
+    - maxMemoryMbytes:               16384          (number, required)
+    - maxParallelCrawlers:           10             (number, required)
+    - isActorBanned:                 null           (string, nullable) # TODO: review
```

Comment:
I am not 100% sure that we should be showing this in API

Reply:
with new billing we'll have to redo this, perhaps we can keep it until then